doc/plugin_intro.rst

```diff
@@ -82,6 +89,15 @@ You must further tell PennyLane about the operations and observables that your d
 
 For a better idea of how to best implement :attr:`~.Device.operations` and :attr:`~.Device.observables`, refer to the two reference plugins.
 
+.. note::
+
+    When writing a plugin device for PennyLane, make sure that your plugin
+    supports as many of the PennyLane built-in operations as possible.
+
+    If the convention differs between the built-in PennyLane operation
+    and the corresponding operation in the targeted framework, ensure that the
+    conversion between the two conventions takes places automatically
```

Comment:
ensure that the conversion between the two conventions takes places automatically